components/end/actions/end-workflow/end-workflow.mjs

```diff
@@ -0,0 +1,18 @@
+export default ({
+  name: "End Workflow",
+  description: "Terminate the workflow execution.",
+  version: "0.0.1",
+  type: "action",
+  key: "end_workflow",
```

Comment:
the component key should include the app slug and should use `-` as a delimiter instead of `_`.